plugin/html5_examples/html5_hud.lua

```diff
@@ -0,0 +1,125 @@
+
+local Window = stingray.Window;
+local Application = stingray.Application;
+local World = stingray.World;
+local WebView = stingray.WebView;
+
+local web_page_url = "http://www.google.com"
+local web_view_material_resource_name = "html5_resources/web_view_2d"
+
+Html5ExampleHudViewport = class(Html5ExampleHudViewport)
+
+function Html5ExampleHudViewport:init(id, editor, window)
+    self._id = id
+    self._editor = editor
+    self._window = window
+
+    self._world = Application.new_world(stingray.Application.DISABLE_APEX_CLOTH)
+    self._viewport = Application.create_viewport(self._world, "default")
+    self._gui = World.create_screen_gui(self._world, "immediate")
+    self._light = self._editor:create_preview_light(self._world)
+    self._shading_environment = World.create_shading_environment(self._world)
+    self._editor_camera = EditorCamera.create_viewport_editor_camera(self._id, self._world, {"QuakeStyleMouseLook", "MayaStylePan"})
+    self._camera = self._editor_camera._camera
+end
+
+function Html5ExampleHudViewport:shutdown()
+    WebView.destroy(self._web_view)
```

Comment:
Make sure to release any web view resources when destroying your window/viewport.